ui/lib/core/addon/components/confirm/message.js

```diff
@@ -0,0 +1,47 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import layout from '../../templates/components/confirm/message';
+
+/**
+ * @module Message
+ * `Message` components trigger and display a confirmation message. They should only be used within a `Confirm` component.
+ *
+ * @example
+ * ```js
+ * <div class="box">
+ * <Confirm as |c|>
+ *   <c.Message
+ *     @id={{item.id}}
+ *     @triggerText="Delete"
+ *     @message="This will permanently delete this secret and all its vesions."
```

Comment:
Sorry that typo again! I've noticed it maybe one more time somewhere ^ there also, maybe just do a search for `vesions` and so you catch them all. (Won't comment on this one again promise! 😄 )

Reply:
whew, good eye! thanks!